### PR TITLE
CIRC-378 Add staff slips context to check-in response body

### PR DESCRIPTION
**Pull request by dmytrotur**
## Purpose
Resolves: [CIRC-378](https://issues.folio.org/browse/CIRC-378)
Add staff slips context to check-in response body

## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [x] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [x] There are no breaking changes in this PR.

### CODE REVIEW

File: ramls/check-in-by-barcode-response.json [hunk 1]
```diff
@@ -99,6 +99,10 @@
         }
       },
       "additionalProperties": false
+    },
+    "staffSlipContext": {
```
Comment (kostyantyn-kh) [inline ramls/check-in-by-barcode-response.json]: please update raml and descriptor minor versions

Comment (dmytrotur) [reply ramls/check-in-by-barcode-response.json]: Done

**Comment by OleksiiMaksymov** [comment]
@dmytrotur 
Based on the description of the history for the tokens `item.enumeration` and `item.chronology` there is a condition `item or holdings, most granular`. 
In the current implementation of `Item.class`, only the representation of the item is used without considering the holding presentation. 
Perhaps this requirement is not included in the scope of this story or need clarification from the PO.

**Comment by dmytrotur** [comment]
@OleksiiMaksymov Thanks for pointing this, I'll contact Darcy on this question.

**Pull request merged**